pkgs/tools/misc/bashcards/default.nix

```
@@ -0,0 +1,30 @@
+{ fetchFromGitHub
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "bashcards";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "rpearce";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1hcdinyfffnaxh7kd6h4fz749llclh4biaj783v9vs53aklghpks";
+  };
+
+  phases = "unpackPhase installPhase";
```

Drop this line. Overriding phases is not encouraged. Let's use this instead to skip the build phase:

```suggestion
  dontBuild = true;
```

---

Got it. Will do.